cni/pkg/install-cni/pkg/install/install.go

```diff
@@ -125,13 +127,14 @@ func (in *Installer) Cleanup() error {
 		}
 	}
 
+	// TODO: why does this delete only the files in the primary target dir and not the secondary?
```

Comment:
Can someone confirm that this is a bug?

Reply:
Yeah I think we should delete it from the secondary as well.

Reply:
Okay, changed this PR into a bug fix. Cleanup will now remove the binaries from the secondary bin dir. Actually, there is no longer a primary and secondary target dir, but a single targetDirs array.